Error while compiling 1.0.1

Yeah so for some reason key-names doesn't compile for me:

```
   Compiling key-names v1.0.1 (https://github.com/HactarCE/key-names#e001bde6)
error[E0308]: mismatched types
   --> /home/cozygalvinism/.cargo/git/checkouts/key-names-cd5ddcb8fe8a0903/e001bde/src/linux.rs:372:9
    |
372 |         &conn,
    |         ^^^^^ expected struct `xcb::base::Connection`, found struct `xcb::Connection`
    |
    = note: expected reference `&xcb::base::Connection`
               found reference `&xcb::Connection`
    = note: perhaps two different versions of crate `xcb` are being used?

error[E0308]: mismatched types
   --> /home/cozygalvinism/.cargo/git/checkouts/key-names-cd5ddcb8fe8a0903/e001bde/src/linux.rs:382:59
    |
382 |     let device_id = xkb::x11::get_core_keyboard_device_id(&conn);
    |                                                           ^^^^^ expected struct `xcb::base::Connection`, found struct `xcb::Connection`
    |
    = note: expected reference `&xcb::base::Connection`
               found reference `&xcb::Connection`
    = note: perhaps two different versions of crate `xcb` are being used?

error[E0308]: mismatched types
   --> /home/cozygalvinism/.cargo/git/checkouts/key-names-cd5ddcb8fe8a0903/e001bde/src/linux.rs:386:47
    |
386 |     Ok(xkb::x11::keymap_new_from_device(&ctx, &conn, device_id, 0))
    |                                               ^^^^^ expected struct `xcb::base::Connection`, found struct `xcb::Connection`
    |
    = note: expected reference `&xcb::base::Connection`
               found reference `&xcb::Connection`
    = note: perhaps two different versions of crate `xcb` are being used?

For more information about this error, try `rustc --explain E0308`.
error: could not compile `key-names` due to 3 previous errors
```

This is on Arch Linux.

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -21,6 +21,6 @@ winapi = { version = "0.3.9", features = ["winuser"] }
 
 [target.'cfg(target_os = "linux")'.dependencies]
 thiserror = "1.0"
-xkbcommon = { version = "0.4", features = ["x11", "wayland"] }
+xkbcommon = { version = "0.4.1", features = ["x11", "wayland"] }
 xcb = "0.9.0"
 wayland-client = "0.29"

--- a/Cargo.toml
+++ b/Cargo.toml
@@ -22,5 +22,5 @@ winapi = { version = "0.3.9", features = ["winuser"] }
 [target.'cfg(target_os = "linux")'.dependencies]
 thiserror = "1.0"
 xkbcommon = { version = "0.4", features = ["x11", "wayland"] }
-xcb = "0.8.0"
+xcb = "0.9.0"
 wayland-client = "0.29"

